### PR TITLE
many changes made to many files but got the liked songs to display on…

### PR DESCRIPTION
**Pull request by mrbrantley**
… user profiles and other user profiles

### CODE REVIEW

**Review by archae0pteryx — APPROVED** [review]
Need a code formatter...

File: app/javascript/components/pages/UserProfile.js [hunk 1]
```diff
@@ -1,24 +1,88 @@
 import React, { Component } from "react";
-import { fetchProfile } from "../api";
+import { thead, tr, th, tbody, Table } from "reactstrap";
```
Comment (sjproctor) [inline app/javascript/components/pages/UserProfile.js:2]: I can see you getting Table from reactstrap, but thead, tr, th, tbody are all regular HTML tags. Do they need to be imported?

**Review by sjproctor — APPROVED** [review]
Looks good, just a couple thoughts. And make sure you are removed commented out code.

File: app/javascript/components/pages/UserProfile.js (hunk 1 above)
Comment (mrbrantley) [reply app/javascript/components/pages/UserProfile.js:2]: Good catch. Will be removed before next update

**Pull request merged**